tests/workflow.test.js

```diff
@@ -0,0 +1,19 @@
+import { Template } from "@exabyte-io/ade.js";
+import { expect } from "chai";
+
+import { ContextProviderRegistry } from "../src/context/registry";
+import { createWorkflows } from "../src/workflows";
+
+// patch to make all context providers available
+Template.providerRegistry = ContextProviderRegistry;
```

Comment:
We should probably patch the template at the top of the wode repo and import from there elsewhere

Reply:
This cleaned itself up nicely